### PR TITLE
Using github issue template forms instead of markdown templates

### LINKED ISSUE #364: Update Issue Templates to redirect questions to the Discussions board
We should update the Issue Template across all our repos with an option to ask a question which would redirect users to https://github.com/open-rmf/rmf/discussions.

We can target to update the templates for [core repos](https://github.com/open-rmf/rmf/blob/main/rmf.repos) first followed by other ones in this org.

### PR DESCRIPTION
**Pull request by aaronchongth**
### Implemented feature

Using github issue template forms instead of markdown templates, following https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms.

Fixes https://github.com/open-rmf/rmf/issues/364 too

See https://github.com/aaronchongth/config/issues/new/choose for how it will look like.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        Describe in a few sentences what the feature consists of and what problem it will solve.
+        Include why you think the feature is necessary, who else will benefit, and why they will benefit.
+        e.g. "When I am ... I want to be able to ... so that I can ..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: implementation-considerations
+    attributes:
+      label: Description
```
Comment (xiyuoh) [inline .github/ISSUE_TEMPLATE/feature_request.yaml]: Noticed that there's a repeated label `Description` here, is this intended?

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    id: steps-to-reproduce-bug
+    attributes:
+      label: Steps to reproduce the bug
+      description: Detailed instructions on how to reliably reproduce the bug.
+      placeholder: |
+        1. Launch RMF using `ros2 launch rmf_demos office_demo.launch`
+        1. Launch rviz using `rviz`
+        1. Add the traffic visualiser
+        1. ...
+        1. See the error displayed
```
Comment (xiyuoh) [inline .github/ISSUE_TEMPLATE/bug_report.yaml]: Should this be 1. 2. 3. instead?

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+      description: If from binaries or docker, give the version. If from source, give the output of git rev-parse HEAD or the repos file you use.
+      placeholder: 2.3.0
+    validations:
+      required: true
+
+  - type: dropdown
+    id: ros-distribution
+    attributes:
+      label: ROS distribution
+      description: State the name of the ROS distribution you are using. Note that we are only support Humble and above, please migrate to a supported distribution if the distribution is not found here.
```
Comment (xiyuoh) [inline .github/ISSUE_TEMPLATE/bug_report.yaml]:
```suggestion
      description: State the name of the ROS distribution you are using. Note that we are only supporting Humble and above, please migrate to a supported distribution if the distribution is not found here.
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 1 above)
Comment (aaronchongth) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: Oops, good catch, it should have been `Implementation Considerations`, https://github.com/open-rmf/.github/pull/4/commits/58745f14d2d14c416a5145151a21ed520911c740

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 2 above)
Comment (aaronchongth) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Markdown would have helped with the numbering, but it we are using a code block :facepalm: thanks for catching that, https://github.com/open-rmf/.github/pull/4/commits/58745f14d2d14c416a5145151a21ed520911c740

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 3 above)
Comment (aaronchongth) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: https://github.com/open-rmf/.github/pull/4/commits/58745f14d2d14c416a5145151a21ed520911c740

**Review by xiyuoh — APPROVED** [review]
LGTM!

**Pull request merged**